endpoints.go

```diff
@@ -59,16 +59,16 @@ func (a sorter) Less(i, j int) bool { return a[i].Name < a[j].Name }
 
 const APIPrefix = "/api/v0"
```

Comment:
We are up to "/api/v1" now, should probably pull this from https://github.com/TRON-US/go-btfs-api/blob/BTFS/request.go#L19 if possible?

Reply:
ok.